frontend/internal-packages/agent/src/langchain/agents/qaGenerateUsecaseAgent/prompts.ts

```diff
@@ -0,0 +1,68 @@
+- No extra text or comments
+- requirementType: Either "functional" or "non-functional" depending on the requirement type
+- requirementCategory: Exact category name from the provided requirements
+- requirement: The specific requirement text being addressed
+- title: Concise, user-focused use case title describing the main action or scenario
+- description: Detailed narrative of user-system interaction, including user actions, system responses, and different scenarios (success and failure cases)
+- Generate multiple use cases if a single requirement has different user scenarios
+- Be specific and avoid vague descriptions
+
+Guidelines for Use Case Generation:
+- ONLY generate use cases for requirements explicitly provided with actual content
+- Skip empty requirement categories (e.g., empty objects {{}} or empty arrays)
+- Focus on realistic user scenarios and system interactions
+- Write from a user/business perspective, not a testing perspective
+- Describe what the user does and how the system responds
+- Include both successful scenarios and error handling
+- Break down complex requirements into multiple focused use cases
+- Use clear, narrative language that tells a story of user interaction
+- Avoid testing terminology like "verify", "validate", "check"
+
+Example output:
+{{
+  "usecases": [
+    {{
+      "requirementType": "functional",
+      "requirementCategory": "Account Management",
+      "requirement": "Allow users to register new accounts with email and personal information",
+      "title": "User Registration",
+      "description": "A user provides their email address, password, and personal information to register a new account. Upon submission, the system validates the input, creates the user account, sends a confirmation email, and allows login after verification. Invalid inputs (e.g. malformed email) result in error messages."
+    }},
+    {{
+      "requirementType": "functional",
+      "requirementCategory": "Administrative Features",
+      "requirement": "Allow administrators to add, edit, and delete product details",
+      "title": "Administrator Product Management",
+      "description": "An administrator accesses the product management interface and can add new products by filling in required details, edit existing product information, or remove products from the catalog. Each action updates the system immediately and reflects changes in the product catalog."
+    }},
+    {{
+      "requirementType": "non-functional",
+      "requirementCategory": "Performance",
+      "requirement": "Support up to 1000 concurrent users",
+      "title": "High Traffic Load Handling",
+      "description": "During peak hours, up to 1000 users simultaneously access the system. The system maintains responsive performance, with page load times under 3 seconds and no service degradation. Users experience smooth navigation and can complete their tasks without delays or timeouts."
+    }}
+  ]
+}}`
```

Comment:
Q: I thought the use cases would correspond to SQL queries, but do you plan to generate SQL separately?